src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -43,20 +44,24 @@
  */
 public class StreetEdge extends Edge implements BikeWalkableEdge, Cloneable, CarPickupableEdge {
 
-    private static Logger LOG = LoggerFactory.getLogger(StreetEdge.class);
+    private static final Logger LOG = LoggerFactory.getLogger(StreetEdge.class);
 
     private StreetEdgeCostExtension costExtension;
 
     private static final long serialVersionUID = 1L;
 
+    /**
+     * Since the majority of StreetEdge instances don't have any turn restrictions,
+     * we create a global instance of an empty set of turn restrictions.
+     */
+    private static final Set<TurnRestriction> NO_TURN_RESTRICTIONS = Set.of();
+
```

Comment:
```suggestion
```
I think this is just stating the obvious, so using the List.of() is easier to read.